apps/opencs/model/world/commands.cpp

```diff
@@ -171,3 +172,71 @@ void CSMWorld::CloneCommand::undo()
 {
     mModel.removeRow (mModel.getModelIndex (mId, 0).row());
 }
+
+CSMWorld::DeleteNestedCommand::DeleteNestedCommand (IdTree& model,
+                                                    const std::string& id,
+                                                    int nestedRow,
+                                                    int parentColumn,
+                                                    QUndoCommand* parent) :
+    mId(id),
+    mModel(model),
+    mParentColumn(parentColumn),
+    QUndoCommand(parent),
+    mNestedRow(nestedRow),
+    NestedTableStoring(model, id, parentColumn)
+{
+    setText (("Delete nested row in " + mId).c_str());
```

Comment:
Not sure about the wording. This is text the user will see and probably be confused about "nested row".